packages/flutter_tools/templates/ffi_plugin/README.md.tmpl

```diff
@@ -0,0 +1,35 @@
+# {{projectName}}
+
+{{description}}
+
+## Getting Started
+
+This project is a starting point for a Flutter 
+[FFI plugin](https://docs.flutter.dev/development/platform-integration/c-interop),
+a specialized package that includes native code directly invoked with Dart FFI.
+
+## Project stucture
+
+This template uses the following structure:
+
+* `src`: Contains the native source code, and a CmakeFile.txt file for building
+  that source code into a dynamic library
+
+* `lib`: Contains the Dart code that defines the API of the plugin, and which
+  calls into the native code using `dart:ffi`
+
+* platform folders (`android`, `ios`, `windows`, etc.): Contains the build files
+  for including the native code library with the platform application.
```

Comment:
i would encourage you to add a LOT more detail here, e.g. pointing to each build system, explaining how it gets invoked, talking about the pubspec.yaml magic, explaining some common patterns, etc.